infra/app/app-config/main.tf

```diff
@@ -14,7 +14,16 @@ locals {
   # 2. Each environment's config will have a database_config property that is used to
   #    pass db_vars into the infra/modules/service module, which provides the necessary
   #    configuration for the service to access the database
+
+  # TODO: #ISSUE_HERE: Move has_database into database_config, below
```

Comment:
Will update with issue number (and create the issue) after the PR is approved and before merging!